examples/http_server/client/src/main.rs

```diff
@@ -68,6 +69,12 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         .await
         .expect("Error while awaiting response");
 
-    log::info!("Got response: {:?}", resp);
+    log::info!("response: {:?}", resp);
+    log::info!(
```

Comment:
Does not the previous line already print the body of the response too?

Reply:
No, it just prints `Body(stream)` or something like that.